lib/twitter_cldr/js/mustache/data_bundle.coffee

```diff
@@ -0,0 +1,38 @@
+###
+// Copyright 2012 Twitter, Inc
+// http://www.apache.org/licenses/LICENSE-2.0
+// TwitterCLDR (JavaScript) v{{version}}
+// Authors:     Cameron Dutro [@camertron]
+                Kirill Lashuk [@KL_7]
+                portions by Sven Fuchs [@svenfuchs]
+// Homepage:    https://twitter.com
+// Description: Provides date, time, number, and list formatting functionality for various Twitter-supported locales in Javascript.
+###
+
+{{#source_map}}
+###
+//@ sourceMappingURL={{locale}}.map
+###
+{{/source_map}}
+
+# required alias definition that adds twitter_cldr to Twitter's static build process
+###<<module_def>>###
+
+
+TwitterCldrDataBundle =
+{{{contents}}}
+
+root = if exports?
+  exports
+else
+  this.TwitterCldrDataBundle = {}
```

Comment:
you're right, it would be possible to do it in node, but not in a browser. We'd have to come up with a clever solution if we want both automatic setting and dynamically changing locale.